addon/templates/components/lazy-render-wrapper.hbs

```diff
@@ -3,8 +3,10 @@
 		passedPropertiesObject=passedPropertiesObject
 		class=class
 		id=id
+		classNames=classNames
+
 		_shouldShowOnRender=_shouldShowOnRender
-    classNames=classNames
+		_isInProcessOfShowing=_isInProcessOfShowing
```

Comment:
My thoughts are that properties should be private if the user never needs to modify them to use the addon. In this case a user should never modify _isInProcessOfShowing so I think it should be private.